src/explore-education-statistics-api-docs/README.md

```diff
@@ -21,12 +21,45 @@ Ruby and its required gems:
 ```shell
 sudo apt install build-essential zlib1g-dev libssl-dev libyaml-dev
 ```
+To run the API docs locally for the first time, install Ruby 3.2.6 currently required by the API Docs App as per: `/explore-education-statistics/src/explore-education-statistics-api-docs/.ruby-version`. 
+The following the instructions below describe using [rbenv](https://github.com/rbenv/rbenv) to install Ruby 3.2.6 on Ubuntu (if you're on windows, use `wsl`!):
+
+1. Download Ruby tarball 3.2.6 from https://www.ruby-lang.org/en/news/2024/10/30/ruby-3-2-6-released/
+  Unzip and go to that location
+    ```shell
+    tar -xzvf ruby-3.2.6.tar.gz
+    cd ruby-3.2.6
+    ```
+
+2. install ruby 3.2.6 version
+    ```shell
+    ./configure --prefix=$HOME/.rbenv/versions/3.2.6
+    make
+    make install
+    ```
+
+3. Add Ruby Version to rbenv
+    ```shell
+    rbenv rehash
+    rbenv global 3.2.6
+    ```
+
+4. Confirm you have the right ruby version
+    ```shell
+    ruby -v
+    ```
+
+5. Install Bundler
+    ```shell
+    gem install bundler
+    rbenv rehash
+    ```
```

Comment:
Is this necessary? Fairly sure that Bundler is already installed with modern Ruby versions. From the Bundler docs:

> This guide assumes that you have [Ruby](https://www.ruby-lang.org/en/downloads/) installed. If you do not have Ruby installed, do that first and then check back here! Any modern distribution of Ruby comes with Bundler preinstalled by default.

https://bundler.io/guides/getting_started.html#getting-started